explorer/src/serializers/operatorsSerializer.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Serializer as JSONAPISerializer,
+  SerializerOptions,
+} from 'jsonapi-serializer'
+import { JobRun } from '../entity/JobRun'
+import { BASE_ATTRIBUTES } from './chainlinkNodeSerializerlizer'
+
+const jobRunsSerializer = (runs: JobRun[], runCount: number) => {
```

Comment:
I feel like this should be attached as a `toJSON` method on the models we want to serialize. `JSON.stringify` automatically calls this method when used on an object that has it defined.

Reply:
We should have serialization tests too.